### DISCUSSION TITLE
I want to add the command "help" to my CLI (as a command and not an option)

### DISCUSSION THREAD

**Discussion by ismailbenhallam** [fastapi/typer#833] [Questions] [answered]
### First Check

- [x] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the Typer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Typer" and didn't find any information.
- [X] I already read and followed all the tutorials in the docs and didn't find an answer.
- [X] I already checked if it is not related to Typer but to [Click](https://github.com/pallets/click).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
import typer

app = typer.Typer()

@app.command(name="help")
def additional_help(ctx: typer.Context):
    print(ctx.help())

if __name__ == "__main__":
    app()
```

### Description

I expect to have a way to get the generated help for all commands (using --help) from the typer.Context

### Operating System

macOS

### Typer Version

0.12.3

### Python Version

3.11.8

**Comment by xandaaah** [comment]
You have to add a second command, as if there is only one, you will not have to do `python app.py command`, just `python app.py`. Typer will do this to ease the creation of single-command apps.

**Reply by ismailbenhallam** [reply]
I have a default command that I want to execute when I run `python app.py`. What I really need is showing the help when I run `python app.py help`

**Reply by ismailbenhallam** [reply, marked as answer]
I finally figured it out.
When I did this:
```python
@app.command(name="help")
def help(ctx: typer.Context):
    print(ctx.get_help())
```
I was getting the help for this command which is "help".
So I had to go back one level (using `context.parent`) to get what I needed:

```python
@app.command(name="help")
def help(ctx: typer.Context):
    print(ctx.parent.get_help())
```
